### ISSUE TITLE
NoSlowdown?

### ISSUE THREAD

**Issue by ItsMennyo** [EthanTG/AntiAura#35]
Info:
- Type: Suggestion

Description:
-  Add NoSlowdown check options in the config.

**Comment by EthanTG** [comment]
This has already been added. And why would this be low priority? I would consider this a priority if it was valid since NoSlow is a major hack.

**Comment by EthanTG** [comment]
@ItsMennyo so you have to be more specific. I'll edit it for you.

**Comment by joehot200** [comment]
Just to clarify, NoSlowdown is built into other checks.

- Eating while walking too fast is blocked by the FastEat check.
- Running while blocking is blocked by the Speed check.
- Shooting an arrow while running at full speed is blocked by the AutoBow check.

**Comment by joehot200** [comment]
LowVelocityAllowedSpeed: 5.5
  MidVelocityAllowedSpeed: 3.0
  HighVelocityAllowedSpeed: 1.5

Have been added

**Comment by joehot200** [comment]
MaxEatingTravelSpeed already exists and is basically the NoSlowdown option for FastEat.

**Comment by ItsMennyo** [comment]
👍

**Comment by joehot200** [comment]
Am closing as there appears to be enough config options for NoSlowdown blockage.
